src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,113 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+import org.apache.commons.lang.ObjectUtils;
+
 public class ArrayList<T> implements List<T> {
+    private static int capacity = 10;
```

Comment:
make it a constant

Reply:
this value changes during program execution